Readme.md

```diff
@@ -1 +1,175 @@
 Webpack loader for nunjucks templates
+
+# shiny-nunjucks-loader
+
+Configurable loader for bundling nunjucks templates with webpack.
+
+## Advantages
+
+- Fully configurable
+- Relative paths to views (see Relative paths section)
+- Recompiles on partials and macros changes
+
+## Quick Start
+
+This loader returns raw HTML. In most cases you would pair it with [html-loader](https://github.com/webpack-contrib/html-loader) so your images and other assets will be processed by webpack.
+
+Install nunjucks and loader with your favourite package manager:
+
+```shell
+# yarn
+yarn add --dev nunjucks shiny-nunjucks-loader html-loader
+
+# npm
+npm install --save-dev nunjucks shiny-nunjucks-loader html-loader
+```
+
+Then add it to your webpack config.
+
+```js
+{
+  module: {
+    rules: [
+      { test: /\.(njk|html)$/, use: ["html-laoder", "shiny-nunjucks-loader"] }
+    ];
+  }
+}
+```
+
+## Configuration
+
+It was a bit deceptive to tell that loader is fully configurable. You can configure almost every aspect of nunjucks, but you can't create new Environment since loader creates one inside.
+
+### Filters, Globals, Extensions
+
+You define filters, globals and extensions in separate JavaScript files. Be careful - this files should be able to run in same environment your webpack running. Then specify paths to this files in options. Let's say, that you created all three and put them to `njk-helpers` folder. Then your config would look like that:
+
+```js
+
+{
+  module: {
+    rules: [
+      { test: /\.(njk|html)$/, use: ["html-laoder", {loader: "shiny-nunjucks-loader", options: {
+          filters: path.resolve(__dirname, 'njk-helpers/filters.js'),
```

Comment:
`use: ["html-laoder"`

Same typo and more below. Find and replace throught the document.